.circleci/config.yml

```diff
@@ -1,25 +0,0 @@
-version: 2
```

Comment:
so I removed circleci instead of travis cause I wasnt sure how to do linux testing with circleci 2.0 ... can you help with that? otherwise we could just use travis for linux testing?